anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
It's exactly the same way it works with text: {{c1::Bob}} is {{c2::your uncle}} => "[...] is your uncle" prompts Bob, and "Bob is [...]" prompts your uncle.
Now you imagine you're blind or that you have a better auditive memory, and that's a use case your change is preventing.